Makefile

```diff
@@ -65,6 +65,6 @@ enterprise-prepare-build: ## Create ./imports/enterprise.go, to link enterprise
 	fi
 
 install-tools:
-	# Try install for go 1.16+, fallback to get
+	# Try install for go 1.17+, fallback to get
```

Comment:
This change is not really needed, as it refers to the usage of `go install` instead of `go get`